docs/examples/frameworks/jax/jax-basic_example.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<p>First step is to __ a pipeline definition...</p><p>add? write? implement?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4937/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done</p>